BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankNetworkingScreenApiCoordinator.swift

```diff
@@ -532,6 +555,27 @@ extension GiniBankNetworkingScreenApiCoordinator: SkontoCoordinatorDelegate {
         }
     }
 
+    private func handleTransactionDocsAlert(on navigationController: UINavigationController,
+                                            extractionResult: ExtractionResult,
+                                            documentId: String?,
+                                            deliveryFunction: @escaping (ExtractionResult) -> Void) {
+        self.handleTransactionDocsAlertIfNeeded(on: navigationController,
+                                                defaultAction: { [weak self] in
+            self?.transactionDocsDataCoordinator?.transactionDocs = []
+            deliveryFunction(extractionResult)
+        },
+                                                attachAction: { [weak self] in
+            if let documentId = documentId {
+                self?.transactionDocsDataCoordinator?.transactionDocs = [.init(documentId: documentId,
+                                                                               fileName: "Document",
+                                                                               type: .document)]
+            } else {
+                self?.transactionDocsDataCoordinator?.transactionDocs = []
```

Comment:
[clarification] @mrkulik do we really need to set to an empty array?

Reply:
For don't attach yes, view is set up to be hidden if there are empty docs
Here it's to handle a case when we don't have documentId. I'm not sure we can face it really, but to be able to handle this case I added this option as default value